gui/src/2D/advancedDynamicTexture.ts

```diff
@@ -25,11 +25,42 @@ import { Color3 } from "babylonjs/Maths/math.color";
 import { WebRequest } from "babylonjs/Misc/webRequest";
 import { IPointerEvent, IWheelEvent } from "babylonjs/Events/deviceInputEvents";
 
+declare var GUIEDITOR: any;
+declare var BABYLON: any;
+
+/**
+ * Interface used to configure the node material editor
+ */
+ export interface IGUIEditorOptions {
+    /** Define the URl to load node editor script */
+    editorURL?: string;
+}
+
 /**
  * Class used to create texture to support 2D GUI elements
  * @see https://doc.babylonjs.com/how_to/gui
  */
 export class AdvancedDynamicTexture extends DynamicTexture {
+    /** Define the Url to load node editor script */
+    public static EditorURL = `https://unpkg.com/babylonjs-gui-editor@${Engine.Version}/babylon.guiEditor.js`;
+
+    private BJSGUIEDITOR = this._getGlobalGUIEditor();
+
+    /** Get the inspector from bundle or global */
+    private _getGlobalGUIEditor(): any {
```

Comment:
will this function work for the es6 package without declaring both `BABYLON` and `GUIEDITOR` on the global namespace? I know a similar approach is used in different areas of the framework, but we will slowly need to move out of global namespace changes.